.gitignore

```diff
@@ -10,3 +10,5 @@ target
 .*.swp
 *.vim
 .ensime*
+.bsp
+.tool-versions
```

Comment:
Used by the [asdf version manager](https://github.com/asdf-vm/asdf) to allow easily toggling Java 8, Java 11, and Java 17. Specifically, I noticed the akka-http project would only build with Java 8, and I'm using 11 and 17 in all other projects.

Reply:
Done